<ui version="4.0" >
 <class>UIToolsEdit</class>
 <widget class="QDialog" name="UIToolsEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tools Editor</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item row="6" column="0" colspan="4" >
    <widget class="QGroupBox" name="gbTool" >
     <property name="title" >
      <string>Tool</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>3</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item row="3" column="2" >
       <widget class="QToolButton" name="tbUpdateWorkingPath" >
        <property name="cursor" >
         <cursor>13</cursor>
        </property>
        <property name="toolTip" >
         <string>Update working path according to file path</string>
        </property>
        <property name="icon" >
         <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/buildclean.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QToolButton" name="tbFilePath" >
        <property name="cursor" >
         <cursor>13</cursor>
        </property>
        <property name="toolTip" >
         <string>Browse for a file, it can be an executable file or a file registered with an application ( ie: html file )</string>
        </property>
        <property name="icon" >
         <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/folder_blue.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLineEdit" name="leFilePath" >
        <property name="toolTip" >
         <string>Command to execute, it can be an executable file or a file registered with an application ( ie: html file )</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" >
       <widget class="QToolButton" name="tbWorkingPath" >
        <property name="cursor" >
         <cursor>13</cursor>
        </property>
        <property name="toolTip" >
         <string>Browse for the tool working path</string>
        </property>
        <property name="icon" >
         <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/folder_green.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2" >
       <widget class="QLineEdit" name="leWorkingPath" >
        <property name="toolTip" >
         <string>Tool working path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLineEdit" name="leCaption" >
        <property name="toolTip" >
         <string>Tool caption</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QToolButton" name="tbFileIcon" >
        <property name="cursor" >
         <cursor>13</cursor>
        </property>
        <property name="acceptDrops" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Browse for an icon file</string>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3" >
       <widget class="QLabel" name="lWorkingPath" >
        <property name="text" >
         <string>Working Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3" >
       <widget class="QLabel" name="lCaption" >
        <property name="text" >
         <string>Caption</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3" >
       <widget class="QLabel" name="lFilePath" >
        <property name="text" >
         <string>File Path</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QToolButton" name="tbHelp" >
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/helpkeyword.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2" >
    <widget class="QDialogButtonBox" name="bbDialog" >
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" >
    <widget class="QPushButton" name="pbNew" >
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>New</string>
     </property>
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/editadd.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="pbDelete" >
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Delete</string>
     </property>
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/editremove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QPushButton" name="pbUp" >
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Up</string>
     </property>
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/buttonup.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QPushButton" name="pbDown" >
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Down</string>
     </property>
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/buttondown.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="Line" name="lMiddle" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item rowspan="6" row="0" column="0" colspan="3" >
    <widget class="QListWidget" name="lwTools" >
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
    </widget>
   </item>
   <item row="5" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>52</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>bbDialog</sender>
   <signal>accepted()</signal>
   <receiver>UIToolsEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>321</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel" >
     <x>172</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbDialog</sender>
   <signal>rejected()</signal>
   <receiver>UIToolsEdit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>321</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel" >
     <x>257</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
